avocado/core/remote/runner.py

```diff
@@ -264,6 +265,13 @@ def run_suite(self, test_suite, variants, timeout=0, replay_map=None):
             self.result.end_tests()
             self.job._result_events_dispatcher.map_method('post_tests',
                                                           self.job)
+            # compare the major version of remote and local version of avocado. If different write to UI and job.log
+            if str(remote_version[0]) != version.MAJOR:
+                log = logging.getLogger("avocado.job")
+                self.job.log.info("WARNING    : avocado version on the remote system is %s, local is %s",
+                                  '.'.join(map(str, remote_version)), version.VERSION)
+                log.warn("WARNING    : avocado version on the remote system is %s, local is %s",
+                         '.'.join(map(str, remote_version)), version.VERSION)
```

Comment:
I see value in displaying the warning before the the tests I'm just not sure if it matters to much. At least in my use case I wouldn't kill a test just because the versions mismatch, but if a test fails the warning message would be nice. Thanks for suggestions, I'll make a new branch with the changes and submit another pull reques! t